enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -152,11 +158,14 @@ func NewProvider(env environment.Env, buildRoot string) (*provider, error) {
 	if err := os.MkdirAll(containersRoot, 0755); err != nil {
 		return nil, err
 	}
-	layersRoot := filepath.Join(buildRoot, "executor", "oci", "layers")
-	if err := os.MkdirAll(layersRoot, 0755); err != nil {
+	imgRoot := *imageCacheRoot
+	if imgRoot == "" {
```

Comment:
this seems like misconfiguration, do you think it'd make more sense to log.Fatal() here?